production/helm/loki/values.yaml

```diff
@@ -813,6 +813,13 @@ tableManager:
   nodeSelector: {}
   # -- Tolerations for table-manager pods
   tolerations: []
+  # -- Configuration for table-manager
+  config:
```

Comment:
This doesn't really have to be in a config block.

Reply:
I rebased and made the changes you requested but the PR is not updating?! Maybe something is wrong with Github? I'll investigate further.

Reply:
GitHub status page is looking colorful. I suspect the problem is on the GitHub side.

https://www.githubstatus.com/

Reply:
Ahh makes sense. I was pulling my hair out on this already :smile: Thanks for the hint!

Reply:
This is now fixed.